doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -165,17 +165,10 @@ The procedure to install CORTX on OVA is mentioned below.
 
     systemctl start|restart <service_name>
     
-#. By default, port 80 may be closed. Run the below mentioned command to open port 80.
-
-   ::
-               
-    salt '*' cmd.run "firewall-cmd --zone=public-data-zone --add-port=80/tcp --permanent"
-    
-    salt '*' cmd.run "firewall-cmd --reload"
-      
 #. Run **ip a l** and record the IP addresses of the following interfaces:
 
-   * ens32 - management 
+   * ens32 - management ip
+   * ens33 - data ip
```

Comment:
ens33 - for public data IP
ens34 -  for private data IP (if there exists one)

Reply:
done